### ISSUE TITLE
Codebox Style Scheme options do not change appearance

### ISSUE THREAD

**Issue by Tyrewt** [giuspen/cherrytree#2192]
Selecting a Style Scheme for Codeboxes doesn't change the color theme. This occurs regardless of which theme option is selected.  I've tested using the Ubuntu Linux, Windows Installer and Portable versions of Cherrytree 0.99.53.0 with the same behavior.

As shown in the screenshot Style Scheme "Tango" is selected yet appearance remains the same as the default cobalt-darkened style.

Is this a bug, or am I misunderstanding the option?

![cherrytree-theme](https://user-images.githubusercontent.com/21961083/211129229-0a98627b-ca9b-425e-9970-17f3dd81b651.PNG)

**Comment by giuspen** [comment]
@Tyrewt in your screenshot I see a white background with selected "tango" in the Code Style Scheme (expected).
If you select "cobalt-darkened" you should have a black background.
Are you saying that you always have a white background no matter what you select?

**Comment by Tyrewt** [comment]
Correct. I realized after posting that Tango may not have been the best example. However, as I mentioned even the default cobalt-darkened is white. Please note that I did change the Text Background to white too under Style Scheme Editor (shown above), but to my understanding the Code style is independent and shouldn't be affected.

Below is a screenshot with the Codebox element selected as cobalt-darkened but the visual style has not changed.

![cherrytree-2](https://user-images.githubusercontent.com/21961083/211200543-27dfcf6f-98cb-44da-ad24-2263d66812c8.PNG)

**Comment by Tyrewt** [comment]
Has this been identified as an issue, or am I mistaken?

**Comment by giuspen** [comment]
I'm a bit puzzled because you are the first to report such issue and I cannot reproduce it.
You are saying that you see exactly the same on Ubuntu, is that right?
Can you run cherrytree from the Ubuntu terminal then and copy/paste the terminal output here? Also a screenshot may help. Also if you attach a test document may help.

**Comment by Tyrewt** [comment]
I am too. Yes, the same occurs across all three (Ubuntu, Windows and Portable). Note I changed default colours under Style Scheme Editor, but that shouldn't matter. The screens below show before and after the codebox was switched to solarized-dark, both existing and a new codebox did not change their respective visual theme.
![ct-2023-01-23 10-52-32](https://user-images.githubusercontent.com/21961083/214090796-2a3d1830-44b1-4da3-993a-a6ba87152526.png)
![ct-2023-01-23 10-52-20](https://user-images.githubusercontent.com/21961083/214090849-8ee97570-fdb1-413c-8632-94457d7b961c.png)

Terminal output:
```
UID@browser:~$ cherrytree
[2023-01-23 10:47:49.988] [   ] [debug] /home/UID/.config/cherrytree/config.cfg parsed
[2023-01-23 10:47:50.042] [   ] [debug] autosave is started
[2023-01-23 10:47:54.036] [   ] [debug] Node name: testing 2
[2023-01-23 10:48:11.983] [   ] [debug] shift images in MenuBar/context menu
[2023-01-23 10:48:50.154] [   ] [debug] autosave: no needs to save file
[2023-01-23 10:49:50.156] [   ] [debug] autosave: time to save file
[2023-01-23 10:50:50.154] [   ] [debug] autosave: time to save file
[2023-01-23 10:51:50.154] [   ] [debug] autosave: no needs to save file
[2023-01-23 10:52:50.166] [   ] [debug] autosave: no needs to save file
```

**Comment by giuspen** [comment]
It must be one of your settings that cherrytree is not digesting well.
Can you do File--Preferences--Export Preferences and zip and attach to this ticket your config.cfg?
You could also help me trying to reset cherrytree to the defaul settings and see if the issue is there already.
You can do that with File--Preferences--Open Preferences Directory, then you:
1) close cherrytree (ensure not in the systray)
2) move all preferences directory content elsewhere (I expect 1 folder and 3 files)
3) start cherrytree
This way cherrytree re-generates from scratch with everything default

**Comment by Tyrewt** [comment]
Which is very odd, because I didn't imported a config to any of the instances (Linux, Windows VM and Win Portable on USB), yet all three are affected. Unless I am doing something wrong.
 
1. The Preferences dir has 1 folder named styles along with 2 files cherrytree.log and config.cfg (attached as *.txt).
2. Testing after reset results in the same behavior.  Code default is cobalt-darkened, switching to lighter theme such as solarized-light or Tango does not change the style.

![Screenshot from 2023-01-24 14-37-54](https://user-images.githubusercontent.com/21961083/214395218-874b5250-493f-40c5-946b-1c8a4987e9e4.png)

[config_2023.01.24_14.26.14.txt](https://github.com/giuspen/cherrytree/files/10493535/config_2023.01.24_14.26.14.txt)

**Comment by giuspen** [comment]
Mhmm... from your latest screenshot, the style of the codebox **is** changed compared to the first screenshot. That is visible from the background that is dark and the text that is light while previously was the other way round.
I wonder if what you mean with style is in fact the syntax highlighting?
What syntax highlighting have you applied to the codebox?
For example in the screenshot below it is python:
![image](https://user-images.githubusercontent.com/6842059/214680893-4a43c14b-bb31-4c75-a27f-6ed4c791cce6.png)

**Comment by Tyrewt** [comment]
That is because we reset the confg which by default is black background. In the first example I had changed default colours under Style Scheme Editor to a white background, I intentionally included a screenshot showing the preferences  and noted that change in subsequent posts.

Having said that, here is another with my original config showing Code changed to "Oblivion", the expect visual appearance should be as show in [Figure 9.07](https://giuspen.com/cherrytreemanual/). Playing around a bit more, I discovered how to get the expected result.

I was changing the "Code" style, what I needed was to change the "Plain Text" style. As shown in this screenshot the codebox style has visually displayed as "Oblivion". Sorry for wasting your time this was a misunderstanding on my part.

![Screenshot from 2023-01-25 16-52-46](https://user-images.githubusercontent.com/21961083/214699916-0e523568-610e-4eaa-ad13-6fa40eb0b206.png)
